libs/cdk/internals/scheduler/src/lib/scheduler.ts

```diff
@@ -364,33 +289,13 @@ function scheduleCallback(
   return newTask;
 }
 
-function pauseExecution() {
-  isSchedulerPaused = true;
-}
-
-function continueExecution() {
-  isSchedulerPaused = false;
-  if (!isHostCallbackScheduled && !isPerformingWork) {
-    isHostCallbackScheduled = true;
-    requestHostCallback(flushWork);
-  }
-}
```

Comment:
same goes for them, I guess they are connected

Reply:
Same question.